pillar/environment-preview-public.sls

```diff
@@ -21,3 +21,21 @@ journal:
     # web_users: # see builder-private
     feature_xpub: false
     submit_url: https://reviewer.elifesciences.org/login
+    calls_to_action:
+      - id: survey2021
+        path: ^/
```

Comment:
This allows us to specify a rule for which pages we wish to apply this call to action to. This regex will match all paths so this should appear on all pages of the journal.